widget_driver/lib/src/utils/runtime_environment_info.dart

```diff
@@ -13,6 +13,11 @@ class RuntimeEnvironmentInfo {
 
   /// Returns `true` if the current runtime is in a test environment and tests are running.
   bool isRunningInTestEnvironment() {
-    return _platformEnvironment.containsKey(_flutterTestEnvKey);
+    bool isRunningInTest = false;
+    assert(() {
+      isRunningInTest = _platformEnvironment.containsKey(_flutterTestEnvKey);
+      return true;
+    }());
+    return isRunningInTest;
```

Comment:
I added the assert here also, since in the DrivableWidget code I only depend on the interface of RuntimeEnvironmentInfo.. So I need to have this code in both places.
If I would only do the "preprocessor" code here, then the DrivableWidget could not know on compile time that the `isRunningInTest` is always gonna be false